server/etcdserver/server.go

```diff
@@ -2484,6 +2485,46 @@ func (s *EtcdServer) updateClusterVersion(ver string) {
 		)
 	}
 
```

Comment:
Might as well rename this method to `updateClusterVersionV2`, to be clear?

Reply:
Can we also update the logging

```
"updating cluster version using v2 API
```

Reply:
```
"setting up initial cluster version using v2 API"
```

Reply:
Added in the next revision as your comments. PTAL @gyuho